pytissueoptics/scene/geometry/bbox.py

```diff
@@ -77,5 +80,74 @@ def yLim(self) -> List[float]:
     def zLim(self) -> List[float]:
         return self._zLim
 
+    @property
+    def xWidth(self):
+        return self.xMax - self.xMin
+
+    @property
+    def yWidth(self):
+        return self.yMax - self.yMin
+
+    @property
+    def zWidth(self):
+        return self.zMax - self.zMin
+
+    def getAxisWidth(self, axis: str) -> float:
+        if axis == "x":
+            return self.xWidth
+        elif axis == "y":
+            return self.yWidth
+        elif axis == "z":
+            return self.zWidth
+
+    def getAxisLimit(self, axis: str, limit: str) -> float:
+        return self._xyzLimits[self._axisKeys.index(axis)][self._limitKeys.index(limit)]
+
+    def getAxisLimits(self, axis: str) -> List[float]:
+        return self._xyzLimits[self._axisKeys.index(axis)]
+
+    def change(self, axis: str, limit: str, value: float):
+        self._xyzLimits[self._axisKeys.index(axis)][self._limitKeys.index(limit)] = value
+        self._checkIfCoherent()
+
+    def changeToNew(self, axis: str, limit: str, value: float) -> 'BoundingBox':
+        newBbox = deepcopy(self)
+        newBbox.change(axis, limit, value)
+        newBbox._checkIfCoherent()
+        return newBbox
+
+    def newFrom(self):
+        newBbox = deepcopy(self)
+        return newBbox
```

Comment:
new from what ? I think you mean .copy() like we used elsewhere in the code base (Vector.copy()). We need to stay coherent.

Reply:
yes